api/src/main/java/io/opentelemetry/common/ReadableAttributes.java

```diff
@@ -21,6 +21,15 @@
  *
  * <p>See {@link Attributes} for the public API implementation.
  */
-public interface ReadableAttributes extends ReadableKeyValuePairs<String, AttributeValue> {
-  AttributeValue get(String key);
+public interface ReadableAttributes {
```

Comment:
I don't quite follow, why so much `extends` clauses were eliminated?

Reply:
They weren't adding any value, and making it much more complicated to deal with the generic keys. It ended up being much simpler and easier to understand without the extra interface that wasn't a meaningful one to the end-consumer.